Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -432,7 +432,8 @@ - (void)launchApplicationAndExecuteTestsWithParser:(BPTreeParser *)parser andCom
     // These are appended by Xcode so we do that here.
     [commandLineArgs addObjectsFromArray:@[
                                            @"-NSTreatUnknownArgumentsAsOpen", @"NO",
-                                           @"-ApplePersistenceIgnoreState", @"YES"
+                                           @"-ApplePersistenceIgnoreState", @"YES",
+                                           @"-XCTIDEConnectionTimeout", @180
```

Comment:
Is it work making this configurable?

Reply:
User shouldn't need to worry about this timeout. This should be managed by bluepill.